src/malloc_io.c

```diff
@@ -760,6 +760,71 @@ malloc_printf(const char *format, ...) {
 	va_end(ap);
 }
 
+static ssize_t
+malloc_write_fd_syscall(int fd, const void *buf, size_t count) {
```

Comment:
`malloc_write_fd_syscall` is still just a sycall wrapper? Why is `inline` also removed for it?